db/migrate/20150609193512_create_songs.rb

```diff
@@ -0,0 +1,14 @@
+class CreateSongs < ActiveRecord::Migration
+  def change
+  	enable_extension 'uuid-ossp'
```

Comment:
Once you've enabled the extension once, you don't have to do it again in subsequent migrations :+1: